build/fluent_loader.js

```diff
@@ -1,16 +1,8 @@
-const { FluentBundle } = require('fluent');
+const { FluentResource } = require('fluent/compat');
 const fs = require('fs');
 
-function toJSON(map) {
-  return JSON.stringify(Array.from(map));
-}
-
-function merge(m1, m2) {
```

Comment:
In Fluent, localization works best when languages are fully isolated. Rather than merging the messages, I suggest that for non en-US locales, two `FluentResource` are included in the dist file. This ensures that inerpolations of strings within other strings are consistently translated, as well as that the formatting of numbers and dates corresponds to the language of the text around them. I implemented this approach in this PR.